05-setting-up.Rmd

```diff
@@ -9,35 +9,169 @@ ottrpal::set_knitr_image_path()
+- `01-make-heatmap.Rmd` - The script that needs to be run second and is also called by run_analysis.sh
+- `README.md` - The document that has the information that will orient someone to this project
+- `plots` - A folder of plots and resulting images
+- `results` - A folder of results
+- `raw-data` - Data files as they first arrive and **nothing** has been done to them yet
+- `processed-data` - Data that has been modified from the raw in some way
+- `util` - A folder of utilities that never needs to be called or touched directly unless troubleshooting something
+
+There are lots of ideas out there for organizational strategies. The key is finding one that fits your team and your project. You can read through some of these articles to think about what kind of organizational strategy might work for you and your team:   
+
+- [Reproducible R example](https://github.com/jhudsl/reproducible-r-example)
+- [Jenny Bryan's organizational strategies](https://www.stat.ubc.ca/~jenny/STAT545A/block19_codeFormattingOrganization.html) [@Bryan2021].
+- [Danielle Navarro's organizational strategies](https://www.youtube.com/playlist?list=PLRPB0ZzEYegPiBteC2dRn95TX9YefYFyy) @Navarro2021
+- [Jenny Bryan on Project-oriented workflows](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/)[@Bryan2017].
+- [Data Carpentry mini-course about organizing projects](https://datacarpentry.org/organization-genomics/) [@DataCarpentry2021].
+- [Andrew Severin's strategy for organization](https://bioinformaticsworkbook.org/projectManagement/Intro_projectManagement.html#gsc.tab=0) [@Severin2021].
+- [A BioStars thread where many individuals share their own organizational strategies](https://www.biostars.org/p/821/) [@Biostars2021].
+- [Data Carpentry course chapter about getting organized](https://bioinformatics-core-shared-training.github.io/shell-genomics/07-organization/index.html) [@DataCarpentry2019].
 
 ## Navigate file paths
 
+In point and click apps (called [Graphical User Interfaces (or GUI pronounced like the word gooey)](https://en.wikipedia.org/wiki/Graphical_user_interface) you navigate to files by clicking on folders. But for R programming and other command line interfaces, we navigate and use files by using `file paths`. `File paths` are the series of folders that it takes to get to a file, not unlike a street address.
+
+To make an analogy, if someone asked you directions to a particular building, the directions you would give would be tailored based on where the person asking is located. In other words your directions would be relative to their location.
+
+But file paths can be *relative* or *absolute*.
+
+Your computer can be given directions relative to where you are calling the command in the computer or they can be absolute directions to a file - basically the full directions to that file, regardless of where you might be already on your computer.  
+
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
```

Comment:
```suggestion
```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "A relative path might be from the local neighborhood to johns hopkins, where as a relative path is analogous to a path that could direct you from further away, so state information would also be included"}
```